src/sagas/projects.js

```diff
@@ -47,6 +50,17 @@ export function* changeCurrentProject() {
   yield call(saveCurrentProject, state);
 }
 
+export function* rehydrateProject({payload: {rehydratedProject}}) {
+  const state = yield select();
+  const allProjectKeys = getProjectKeys(state);
+  if (allProjectKeys.includes(rehydratedProject.projectKey)) {
+    yield put(__changeCurrentProject(rehydratedProject.projectKey));
```

Comment:
This isn’t hurting anything but I don’t think it’s really necessary—if anything the copy of the project store in local storage is likely to be _more_ up-to-date than the one stored in Firebase, so we might as well let the local storage version clobber it?